### DISCUSSION TITLE
ARM version or emulation capability

### DISCUSSION THREAD

**Discussion by akamienski** [Open-Shell/Open-Shell-Menu#1968] [General]
### Is your feature request related to a problem? Please describe.

Hello,

I saw few issues opened about ARM verison, but it was quite a log time ago. What's the status on it now, will it still not be compiled to be running on ARM, or was some version already compiled to work on ARM? 

I feel like OSH can easily be running in emulation, but it fails to run, freezing explorer.exe completely when PC is restarted after OSH installation. Is there some kind of fix to that?

### Describe the solution you'd like

Open-Shell working either natively on ARM or emulated

### Area of issue

Installation/Other

### Alternatives you've considered

Windows 11 on ARM

**Comment by ge0rdi** [comment]
ARM64 support is tracked in #553 feature request.
There is #641 pull-request that proposed some changes towards ARM64 version, but it was not finished yet.

I'd not expect any changes in this regard unless somebody would propose better changes or finalize existing PR.
